### PR TITLE
Docupdates

### PR DESCRIPTION
**Pull request by funkyeah**
@raix hopefully pull requesting suitable documentation updates to add more details around Push.send and its parameters. I am not 100% sure that what I am saying is even correct so I think it would be worth a quick read through from your side.

### CODE REVIEW

**Comment by funkyeah** [comment]
also this is my first real pull request to someone else's repo so if I need to do something differently please let me know

File: docs/ADVANCED.md [hunk 1]
```diff
@@ -147,4 +147,62 @@ Please note that `Push.Configure` is called automatically when using the `config
...
+You can send push notifications from the client or the server using Push.send(). If sending from the client you are required to use [allow/deny](ADVANCED.md#client-security)) rules.
+
+There are 4 required parameters that must be passed to `Push.send`. They are:
+* `from` : reserved for future use. intended to be internally used by gcm to generate a collapse key. this can be any random string at the moment
+* `title` : the bold title text that is displayed in the notification
+* `text` : the normal sub-text that is displayed in the notification
+* a selection query from below
+
+The 4th parameter is a selection query for determining who the message should be sent to. This query can be one of the three following items:
+* `query` : {} or {userId : 'XXXXX'} or {id : 'XXXXX'}
```
Comment (alexcorre) [inline docs/ADVANCED.md]: if possible id also do `` around the json objects. like so:
`{}` or `{userId : 'XXXXX'}` or `{id : 'XXXXX'}`

**Comment by alexcorre** [comment]
thanks for doing this. lets hope @raix pulls it in

**Comment by raix** [comment]
Super @funkyeah looks good! Its a huge help to the community - I know too much of the internals and taking "stuff" for granted, so I'm not the best to write docs I think :)

**Comment by raix** [comment]
@alexcorre thanks for review, awesome!

**Pull request merged**